test/e2e/cli_test.go

```diff
@@ -391,7 +391,7 @@ func (s *CLISuite) TestWorkflowDeleteAll() {
 		SubmitWorkflow().
 		WaitForWorkflow(30*time.Second).
 		Given().
-		RunCli([]string{"delete", "--all", "-l", "argo-e2e"}, func(t *testing.T, output string, err error) {
+		RunCli([]string{"delete", "--all"}, func(t *testing.T, output string, err error) {
```

Comment:
this can't be correct - all workflows created by e2e test must get this label - so this probably means there is a bug

Reply:
Could you clarify where is incorrect? There is only one workflow called basic at this point so deleting all would just means only this workflow gets deleted.

Reply:
All test workflows must be annotated with `argo-e2e` - if it fails to delete, then it indicates that `--all` is not deleting all workflows labelled with `argo-e2e`. `--all` should work with `-l`, but does not appear to.

Reply:
Okay I see. I've reverted the flag check in delete command.